.travis.yml

```diff
@@ -2,18 +2,13 @@ language: ruby
 cache: bundler
 sudo: false
 
-rvm:
-  - 2.2.7
-  - 2.3.4
-
```

Comment:
Please don't rip out permutations from the build matrix. These are all here for a reason.  If you think a version or permutation should be dropped, that should be reserved for an entirely separate Issue discussion and PR. It's not appropriate to co-opt a PR with something like this.

Reply:
I'm sorry about that, I just confused this with actual RVM versions 😆 https://github.com/integrallis/stripe_event/pull/94#issuecomment-348333567